src/scss/custom/modules/_imageslider.scss

```diff
@@ -0,0 +1,19 @@
+#homeSlider {
+  .carousel-link {
+    align-items: start;
+    height: 100%;
+  }
+
+  .carousel-content {
+    height: 100%;
+  }
+}
+
+.ratio-homeSlider {
+  min-height: 300px;
+
+  --bs-aspect-ratio: calc(9 / 21 * 100%); // 21x9
+  @include media-breakpoint-up(md) {
+    --bs-aspect-ratio: calc(7 / 30 * 100%); // 30x7
+  }
+}
```

Comment:
Be careful, we already have some _slider partials inside `custom` and `core` scss folders, you should work inside these files or remove them and create _imageslider if you want to stick to module name

Reply:
the file is already called _imageslider  ..... :)

Reply:
should be inside core or custom folder?

Reply:
![image](https://user-images.githubusercontent.com/14963751/144432749-3962a936-5476-43a2-aeb1-541c4280c93c.png)
I think it should be in custom, theme developer will be able to handle his slider styling if he removes the custom part

We already have a _slider partial inside custom, let's move his content into your new _imageslider file ? The _slider is an error I think so

Reply:
Not agree. My _imageslider.scss are ONLY styles for the home slider. Not any other slider in the store ( product, quickview or whatever) 
Thats the reason to put in custom/modules

Reply:
Ok, good arg, but in the future, if we find that some styles should only go to the homeSlider, then we'll need to move these!